addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -40,12 +40,12 @@
                     <button t-if="thread.type !== 'mailbox'" class="btn" t-ref="settings" title="Show Call Settings" t-on-click="toggleSettings">
                         <i class="fa fa-lg fa-gear text-700"/>
                     </button>
-                    <div t-if="store.guest and !env.isSmall" class="d-flex align-items-center">
-                        <img class="o-mail-discuss-guest-avatar mx-1 rounded-circle o_object_fit_cover flex-shrink-0" alt="Avatar" t-att-src="guestAvatarUrl"/>
+                    <div t-if="props.public and !env.isSmall" class="d-flex align-items-center">
+                        <img class="o-mail-discuss-guest-avatar mx-1 rounded-circle o_object_fit_cover flex-shrink-0" alt="Avatar" t-att-src="avatarUrl"/>
                         <AutoresizeInput
                             className="'o-mail-discuss-thread-guest-name lead fw-bold flex-shrink-1 text-dark'"
                             onValidate.bind="renameGuest"
-                            value="store.guest.name"
+                            value="store.guest?.name or store.user?.name"
```

Comment:
functionally, we should only allow guests to change names here and not the user. (like what we did before)

Reply:
Agree, rename of user has more implication, it cannot be simply changed here. Better not allow renaming of user in public page